langwatch/src/components/MainMenu.tsx

```diff
@@ -76,134 +68,66 @@ export const MainMenu = React.memo(
-          <Box width="full" paddingX="6px">
-            <Button
-              asChild
-              className="group"
-              variant="plain"
-              onClick={() => setIsExpanded(!isExpanded)}
-              width="full"
-              minWidth="0"
-              padding={2}
-              paddingRight={isExpanded ? "6px" : "2px"}
-              size="lg"
-              _icon={{
-                width: "auto",
-                height: "auto",
-                maxWidth: "none",
-                maxHeight: "none",
-              }}
-              backgroundColor="white"
-              _hover={{
-                backgroundColor: isExpanded ? undefined : "gray.50",
-              }}
-            >
-              <HStack fontSize="32px" fontWeight="bold" gap={0}>
-                <LogoIcon width={25} height={34} />
-                <Spacer />
-                <Box
-                  padding={isExpanded ? 2 : 0}
-                  borderRadius="md"
-                  backgroundColor="white"
-                  _groupHover={{
-                    backgroundColor: isExpanded ? "gray.50" : undefined,
-                  }}
-                >
-                  {isExpanded ? (
-                    <ChevronLeft size={16} />
-                  ) : (
-                    <ChevronRight size={16} />
-                  )}
-                </Box>
-              </HStack>
-            </Button>
-          </Box>
+          <Center width="full" paddingX="6px">
+            <LogoIcon width={25} height={34} />
+          </Center>
 
           <VStack width="full" height="full" gap={0} align="start">
             <PageMenuLink
               path={projectRoutes.home.path}
               icon={TrendingUp}
               label={projectRoutes.home.title}
               project={project}
-              isExpanded={isExpanded}
             />
             <PageMenuLink
               path={projectRoutes.messages.path}
               icon={MessageSquare}
               label={projectRoutes.messages.title}
               project={project}
-              isExpanded={isExpanded}
             />
 
             <PageMenuLink
               path={projectRoutes.evaluations.path}
-              icon={Shield}
+              icon={CheckSquare}
               label={projectRoutes.evaluations.title}
               project={project}
-              isExpanded={isExpanded}
             />
 
             <PageMenuLink
               path={projectRoutes.workflows.path}
               icon={PuzzleIcon}
               label={projectRoutes.workflows.title}
               project={project}
-              isExpanded={isExpanded}
             />
 
-            <PageMenuLink
+            {/* <PageMenuLink
               path={projectRoutes.promptConfigs.path}
               icon={Layers} // Use an appropriate icon
               label={projectRoutes.promptConfigs.title}
               project={project}
-              isExpanded={isExpanded}
-            />
+            /> */}
```

Comment:
Should this be done with a feature flag instead?

Reply:
probably it should... but we dont have a feature flag functionality yet, if we use an env var we'd need a new deploy anyway so... just uncommenting in a next commit will be quick for now